src/Xrefcheck/CLI.hs

```diff
@@ -47,7 +47,7 @@ modeReadM = eitherReader $ \s ->
 
 data Command
   = DefaultCommand Options
-  | DumpConfig RepoType FilePath
+  | DumpConfig Flavor FilePath
```

Comment:
And for this command it actually makes sense to take `RepoType` rather than `Flavor`. Currently these are equal sets (of repo types and MD flavors), but one day we may support another repo type that may use an existing flavor (e. g. GH-flavored MD). Then `RepoType` will become a different entity and it will be appropriate here. That repo type may also have different `notScanned` and `virtualFiles` for example.

That said, I don't think it's _necessary_ to bother with this stuff _now_, I am just pointing it out. So I think that `RepoType` and `Flavor` are 2 different entities, currently they would be equal and we use `Flavor` to denote `RepoType` also and that's fine for now.

Reply:
Yeah, at least half of me also thinks so.

I guess I'll leave a comment about this distinction and rename the CLI option.